ci/backup_questionnaire_state.yaml

```diff
@@ -8,6 +8,7 @@ params:
   SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
   PROJECT_ID:
   BUCKET_NAME:
+  FILE_PREFIX:
```

Comment:
Decided not to add a default in the end, as it wouldn't be correct when we come to support multiple survey types.